sdk/typescript/src/index.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Mysten Labs, Inc.
```

Comment:
I moved all the parts that have not been deprecated to the bottom of this file so it's easier to track whats left